modules/govuk/templates/etc/cron.d/snapshot.erb

```diff
@@ -0,0 +1 @@
+*/10 0-4,6-23 * * * <%= @snapshot_user %> /usr/local/bin/rummager-snapshot
```

Comment:
Not unresponsive, it will just return a 503 straight away.

I'll take a look at adding a setlock around both tasks. We'll need to change the rummager rake task first to pass the extra param.

For the snapshot cron we'll use `setlock -n` (give up if locked) - this way we can have the cron execute frequently and get multiple processes waiting on the lock.

For the pruning job we can use `setlock -N` (wait for lock).